science/hacktv/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        kesterlester hacktv 0.0.4 v
+github.tarball_from releases
+
+categories          science
+platforms           darwin
```

Comment:
this is the default since MacPorts v2.7 so the `platforms` line should be removed

Reply:
Fixed in 32fca62